docs/INSTALL.md

```diff
@@ -458,6 +458,11 @@ the following:
   to provide a reliable delivery of notifications (e.g.
   `--zmqpubrawblock=tcp://127.0.0.1:28332` and
   `--zmqpubrawtx=tcp://127.0.0.1:28333`).
+- Make sure you do NOT use the bitcoind config setting `--rpcserialversion`.
+  Lightning depends on segwit transactions and the default rpc bitcoind
```

Comment:
We should mention this `-rpcserialversion` must be set to 1 and if we don't explicitly set it in the config file we are fine as it's already default to 1.

Also feel like there's no need to explain in details other than when setting config value to 0 the witness data won't be returned.